map/map_tests/bookmarks_test.cpp

```diff
@@ -1475,7 +1475,7 @@ UNIT_CLASS_TEST(Runner, ExportAll)
     TEST(base::DeleteFileX(indexPath), ());
     TEST(base::DeleteFileX(tmpPath), ());
   };
-  bmManager.PrepareFileForSharing(std::move(categories), checker);
+  bmManager.PrepareFileForSharing(std::move(categories), checker, KmlFileType::Text);
```

Comment:
Does it export KML or KMZ? Previously, it was always KMZ, right?

Reply:
As described above, I removed kmz enum value and `Text` for multiple categories now means `Put all to KMZ`.

Reply:
A comment in the code would be helpful.